CHANGELOG.md

```diff
@@ -61,7 +68,6 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Changed
 - Compatible with Pydantic v1 & v2
-- Add lower pin on `botocore` / `boto3` to help dependency resolution
```

Comment:
nitpick: should we keep this as despite no longer applying in `0.7.0` it was a change that did still happen in `0.6.0`?

Reply:
yeah good point, I can add another change in 0.7.0 saying it's no longer there!